bankayma_account/views/menu.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <menuitem
+        id="menu_bankayma_income"
+        parent="bankayma_base.menu_bankayma_root"
+        name="Income"
+        action="account.action_move_out_invoice_type"
+        sequence="10"
+    />
+
+    <record id="account.action_move_out_invoice_type" model="ir.actions.act_window">
+        <field name="context">{
+            'default_move_type': 'out_invoice',
+            'default_invoice_line_ids': [
+                (0, 0, {'display_type': 'product'}),
+                (0, 0, {'display_type': 'line_note'}),
+            ]
+        }
```

Comment:
@toyrobot this is how you can set defaults, by manipulating the context dict of the action. And the weird notation is actually what https://github.com/OCA/OCB/blob/16.0/odoo/fields.py#L3931 returns for the `create` case, as we can't use the helper here